user/api/serializers.py

```diff
@@ -1,14 +1,76 @@
 from django.contrib.auth import get_user_model
+from django.contrib.auth.password_validation import validate_password
+from django.core.validators import RegexValidator
 from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+from user.api.constants import PASSWORD_NOT_MATCH_ERROR_MSG, FIRST_NAME_CHAR_ONLY_ERROR_MSG, LAST_NAME_CHAR_ONLY_ERROR_MSG, \
+    REGEX_FOR_CHAR_ONLY
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
 
 User = get_user_model()
 
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ["username", "email", "name", "url"]
+        fields = ('username', 'email', 'first_name', 'last_name')
 
+
+class RegisterSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(
+        required=True,
+        validators=[UniqueValidator(queryset=User.objects.all())], max_length=30
+    )
+
+    password = serializers.CharField(write_only=True, required=True, validators=[validate_password])
+    password2 = serializers.CharField(write_only=True, required=True)
+    first_name = serializers.CharField(
```

Comment:
Not necessary first name and last name characters.

Reply:
yes, for the practice purpose added here